src/ashell/shell-state.c

```diff
@@ -658,12 +659,31 @@ void ashell_run_boot_cfg()
         return;
     }
 
+    size_t tssize = strlen(__TIMESTAMP__);
     ssize_t size = fs_size(file);
-    if (size > 0 && size <= 12) {
-        char data[size+1];
-        count = fs_read(file, data, size);
-        data[size] = '\0';
-        ashell_run_javascript(data);
+    // Check that there is something after the timestamp
+    if (size > tssize) {
+        char ts[tssize];
+        count = fs_read(file, ts, tssize);
+        if (count == tssize && strncmp(ts, __TIMESTAMP__, tssize) == 0) {
+            // skip the space
+            fs_seek(file, 1, FS_SEEK_CUR);
```

Comment:
If you add the newline to BUILD_TIMESTAMP (or change it to a space) like my example above, then you don't need to skip a space here (or add it above).